docs/pages/sdk/create-coinbase-wallet-sdk.mdx

```diff
@@ -0,0 +1,37 @@
+import SdkParameters from './snippets/sdk-paramaters.mdx'
+
+# createCoinbaseWalletSDK 
+Creates a new `sdk` object for getting a `CoinbaseWalletProvider` instance. 
+
+## Usage
+:::code-group
+```ts twoslash [provider.ts]
+import { sdk } from './setup'
+
+// Create provider
+export const provider = sdk.getProvider();
+// Use provider
+const addresses = provider.request({method: 'eth_requestAccounts'});
+```
+```ts twoslash [setup.ts] filename="setup.ts"
+// @ts-expect-error - TODO: update when the SDK is released
```

Comment:
Once a new version of the SDK has been published we can remove this

Reply:
let's hold off on merging this PR until SDK release? i think this repo has auto deploy